### PR TITLE
fix: id:47815 calendar component accesibility

### PR DESCRIPTION
**Pull request by Farhan-emgage**
calendar component accessibility

### CODE REVIEW

**Comment by emgageadmin** [comment]
https://atera.tpondemand.com/entity/47815

File: src/components/Calendar/Calendar.css [hunk 1]
```diff
@@ -276,6 +276,11 @@ button.rbc-input::-moz-focus-inner {
       -moz-transition: all .3s ease-in-out;
       -webkit-transition: all .3s ease-in-out;
     }
+
+    .rbc-date-cell.rbc-now > a:focus, .rbc-date-cell.rbc-now.rbc-current > a:focus {
```
Comment (weegrom) [inline src/components/Calendar/Calendar.css:280]: Add each class declaration on it's own line and remove nesting indents.

**Review by weegrom — CHANGES_REQUESTED** [review]
@Farhan-emgage please see my comment. Take a look at https://github.com/emgage/engage-ui/pull/567 first as it provides greater details as to why these changes are being requested.

File: src/components/Calendar/Calendar.css (hunk 1 above)
Comment (Farhan-emgage) [reply src/components/Calendar/Calendar.css]: i removed nesting declaration

Comment (weegrom) [reply src/components/Calendar/Calendar.css]: @Farhan-emgage I see that the declaration are now on their own lines, however the code is still nested. Can you please remove the nesting. Thanks

Comment (Farhan-emgage) [reply src/components/Calendar/Calendar.css]: as your comment i understand that you are asking about i declare css property like ".rbc-date-cell.rbc-now > a:focus". and you want to use it like 
.rbc-date-cell{
     .rbc-now{
          a{
            &:focus{
             }
          }
     }
}

Comment (weegrom) [reply src/components/Calendar/Calendar.css]: @Farhan-emgage by nesting I mean:

.classname {
  styles...
}
    .classname {
      styles...
    }
        .classname {
          styles...
        }

Which is very confusing, as it makes it look like the styles are actually nested as is done is scss, like so:

.classname {
  styles...

    .classname {
      styles...

        .classname {
          styles...
        }
    }
}

If they are not actually nested styles, then they should not appear nested, but should have a flat structure like so:

.classname {
  styles...
}
.classname {
  styles...
}
.classname {
  styles...
}

Comment (weegrom) [reply src/components/Calendar/Calendar.css]: @Farhan-emgage can you please take care of the above request so we can get this merged.

**Review by weegrom — APPROVED** [review]

**Pull request merged**